docs/modules/reference/pages/glossary.adoc

```diff
@@ -21,15 +20,16 @@ Auto-discovered nodes:: Nodes that are automatically added to {page-component-ti
 (See xref:operation:provisioning/auto-discovery.adoc[Auto-Discovery].)
 
 Availability:: The percentage of time a node has been detected as being accessible.
-{page-component-title} typically shows availability statistics for the past 24 hours; this is calculated by subtracting the amount of outage time for a node, interface, or service.
+{page-component-title} typically shows availability statistics for the past 24 hours.
+This is calculated by subtracting the amount of outage time for a node, interface, or service during the time period.
 
 Azure:: Microsoft Azure is a cloud hosting platform that allows users to host a variety of different services.
 
 Bean Scripting Framework (BSF):: A framework that allows scripting to be used in Java code via a set of Java classes.
-{page-component-title} uses BSF-compatible scripts to determine the status of a service.
-(See xref:reference:provisioning/detectors/BsfDetector.adoc[BSFDetector].)
+The {page-component-title} xref:reference:provisioning/detectors/BsfDetector.adoc[BSF Detector] runs arbitrary BSF scripts to determine the existence of a service, and the xref:reference:service-assurance/monitors/BSFMonitor.adoc[BSF Monitor] uses scripts to determine the status of a service.
 
 Cassandra:: An open-source distributed/scale-out NoSQL database designed to hold large amounts of structured or unstructured data across many servers.
+The optional Newts storage strategy requires a Cassandra instance as back-end storage.
```

Comment:
Make "Newts" a link and remove the link below.

Reply:
Done in VSC and will be committed with those changes.